components/sup/src/manager/mod.rs

```diff
@@ -87,7 +87,10 @@ impl Manager {
                                                  None));
         outputln!("Butterfly Member ID {}", server.member_id());
         for peer_addr in gconfig().gossip_peer() {
-            let addr: SocketAddr = peer_addr.parse().unwrap();
+            let addrs: Vec<SocketAddr> = peer_addr.to_socket_addrs()
+                                                  .expect("Unable to resolve domain")
```

Comment:
Doesn't the `expect` mostly take care of this? It conveys how the input failed and if it does fail, I don't see harm in the panic as long as its made clear to the user why.